app/templates/auth/login.html

```diff
@@ -47,7 +47,7 @@ <h3 class="validation-masthead-heading" id="validation-masthead-heading">
     <h1>Supplier login</h1>
 </header>
 
-<form action="{{ url_for('.process_login', next=next) }}" method="POST">
+<form autocomplete="off" action="{{ url_for('.process_login', next=next) }}" method="POST">
```

Comment:
Not totally sure, [but it seemed like the safe way to go](http://stackoverflow.com/questions/2530/how-do-you-disable-browser-autocomplete-on-web-form-field-input-tag#comment4833209_2531).